lib/TWCManager/Vehicle/TeslaAPI.py

```diff
@@ -227,11 +226,12 @@ def apiRefresh(self):
             "refresh_token": self.getCarApiRefreshToken(),
         }
         req = None
+        now = time.time()
         try:
             req = requests.post(self.refreshURL, headers=headers, json=data)
             self.master.debugLog(2, "TeslaAPI", "Car API request" + str(req))
             apiResponseDict = self.json.loads(req.text)
-        except:
+        except requests.exceptions.RequestException:
```

Comment:
This doesn't handle other exceptions that may be thrown by this block

Reply:
`requests` only raises an exception of `RequestException` while `json.loads` can also raise an `ValueError` exception. You are totally right, that change is dumb - my fault. It should have been `except requests.exceptions.RequestException or ValueError:` to be correct and PEP 8 coding style compatible.

Reply:
Sounds good, however are we sure there's no code path with a chance of an ```AttributeError``` if ```req``` is ```None``` due to an issue with the request and we try to reference ```.text```?

Reply:
I really like the accuracy and 'eye for details' you are putting into this project! You are raising a very valid point as the proposed change doesn't handle and malformed nor non-existent `req.text`!

How about the following rewrite: `except requests.exceptions.RequestException or self.json.JSONDecodeError:`? This will handle all possible exceptions from `requests` and `json` (if I understood the corresponding documentations current). Would that change be acceptable?

Reply:
That sounds good to me :)